doc/Customization-guide/Endpoint-SDKs/C-SDK/Porting-guide/index.md

```diff
@@ -4,3 +4,69 @@ title: Porting guide
 permalink: /:path/
 sort_idx: 40
 ---
+
+{% include variables.md %}
+
+* TOC
+{:toc}
+
+
+# Prerequisites
+
+Kaa C SDK can be ported to all devices  with or without any operations sytem. For example it can be ported to: Linux, Windows, Android, IOS, FireFox, Solaris, microcontrollers with or without any Rtos, etc.
+Kaa supports Big and Little Endian platforms automaticaly, user doesn't need to care about it.
+Note! A platform must have enough memory for the SDK and a network controller with TCP-IP stack.
```

Comment:
Not a platform but the target/board. 
Any way, as user I want to see how much memory I need (the minimum configuration). 
For example: 30KB of ROM and 10KB of RAM.